### PR TITLE
Minor CSS fixes for race-room squishing.

### PR DESCRIPTION
**Pull request by tsigma6**
Some minor fixes that should improve experience for desktop users. Please see comment on 7b0302e for rational.

- Race rooms will have a much more consistent look
- Removed media-query that caused excessive width
- Fixed a small issue with pronoun badges.

### CODE REVIEW

**Comment by tsigma6** [comment]
Disregard comments 7b0302e, after additional testing, specifying both a width and the flex-basis prevents the flex resize to compensate for wrapped messages, thus limiting the resizing to only when a new member joins or race info is updated.

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ __pycache__
 /node_modules
 /project/settings/local.py
 /static
+/venv
+/.idea
```
Comment (deains) [inline .gitignore:11]: Found the JetBrains user. 🙂

Please don't add stuff to `.gitignore` unless it's artifacts created by the project itself. IDE files should be exlcuded by your `.gitignore_global` file: https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

**Review by deains — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (tsigma6) [reply .gitignore]: Doh, thanks for the info.

**Review by deains — APPROVED** [review]
Have checked this out on desktop and it's working fine there. However, there are some issues with these changes on mobile. I'll get those patched up myself so no need to adjust. :)

**Pull request merged**